deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,87 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+/*Schrijf hier jouw CSS*/
+
+/* --------- opdracht 1.1 -----------*/
+.list-1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+
+/* --------- opdracht 1.2 -----------*/
+.list-2 {
+    display: flex;
+    flex-direction: row;
+    justify-content:center;
+    align-items: flex-start;
+    flex-basis: 300px;
+}
+
+.longlist1 {
+    flex-basis: 300px;
+}
+
+/* --------- opdracht 1.3 -----------*/
+.list-3 {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+}
+
+.item1 {
+    align-self-: flex-start;
+}
+
+.item2 {
+    align-self: center;
+}
+.item3 {
+    align-self: flex-end;
+}
+
+/* ---------opdracht 1.4------------*/
+
+
+.list-4 {
+    display: flex;
+    flex-direction: row;
+    align-content: space-around;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    gap: 10px;
+}
+
+
+
+/* ---------opdracht 1.5------------*/
+.div1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.column1 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 40px;
```

Comment:
Wat mij betreft prima gedaan. Het had uiteraard met minder code gekund. 
Tip: gebruik bij opdracht 5 de selector nth-of-type(getal) om de elementen onafhankelijk te verplaatsen.